Source/IdentityApi/IdentityApi/Interfaces/IUserLocationManager.cs

```diff
@@ -0,0 +1,22 @@
+﻿using IdentityApi.Models;
+
+namespace IdentityApi.Interfaces
+{
+    public interface IUserLocationManager
+    {
+        /// <summary>
+        /// Checks if ip adress is blacklisted 
+        /// </summary>
+        public Task<bool> IsIPLockedAsync(string ipAddress);
+
+        /// <summary>
+        /// Saves user location
+        /// </summary>
+        public Task<UserLocation> LogLocationAsync(UserLocation location);
```

Comment:
Same comment as above, and for all the other calls.
This one in particular, it's called LogLocation but we are returning something?